CMakeLists.txt

```diff
@@ -1,35 +1,37 @@
 PROJECT(ChimeraTK-ControlSystemAdapter-DoocsAdapter)
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.16)
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
 
 set(${PROJECT_NAME}_MAJOR_VERSION 01)
-set(${PROJECT_NAME}_MINOR_VERSION 07)
-set(${PROJECT_NAME}_PATCH_VERSION 01)
+set(${PROJECT_NAME}_MINOR_VERSION 08)
+set(${PROJECT_NAME}_PATCH_VERSION 00)
 include(cmake/set_version_numbers.cmake)
 
 include(cmake/set_default_build_to_release.cmake)
 include(cmake/set_default_flags.cmake)
 
-# linker flags
+include(cmake/format_options.cmake)
+# global linker flags. Note, this is not exported.
 set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--export-dynamic")
 
-include(cmake/add_dependency.cmake)
 include(cmake/enable_code_coverage_report.cmake)
 include(cmake/enable_code_style_check.cmake)
 
-add_dependency(ChimeraTK-ControlSystemAdapter 02.05 REQUIRED)
-add_dependency(doocs-server-test-helper 01.05 REQUIRED)
-add_dependency(DOOCS 22.05.0 COMPONENTS server REQUIRED)
+FIND_PACKAGE(ChimeraTK-ControlSystemAdapter 02.06 REQUIRED)
+
+FIND_PACKAGE(doocs-server-test-helper 01.06 REQUIRED)
+
+FIND_PACKAGE(DOOCS 22.11 COMPONENTS server REQUIRED)
 
 FIND_PACKAGE(Boost COMPONENTS thread REQUIRED)
 
-# libXML++ is used for parsing/writing XML files
 FIND_PACKAGE(PkgConfig REQUIRED)
+# note, libxml++ is already used in ControlSystemAdapter but set as PRIVATE. 
+# It makes sinse to keep versions in sync.
 set(LIBXML++_VERSION "libxml++-2.6")
-PKG_CHECK_MODULES(LibXML++ REQUIRED ${LIBXML++_VERSION})
-PKG_CHECK_MODULES(glib REQUIRED glib-2.0)
-include_directories(SYSTEM "${LibXML++_INCLUDEDIR}/${LIBXML++_VERSION}" ${glib_INCLUDE_DIRS} ${LibXML++_INCLUDE_DIRS})
+PKG_CHECK_MODULES(LibXML++ REQUIRED IMPORTED_TARGET ${LIBXML++_VERSION})
+PKG_CHECK_MODULES(glib REQUIRED IMPORTED_TARGET glib-2.0)
```

Comment:
Similar to the backend, I think there is no need anymore for explicitly looking for glib

Reply:
done